types/nearley/index.d.ts

```diff
@@ -35,10 +25,22 @@ export class Parser {
      */
     results: any[];
 
+    constructor(rules: CompiledRules, start: string, options?: ParserOptions);
```

Comment:
This constructor is deprecated, can we remove it?

Reply:
Didn't know about that, it's not mentioned in the source. Will remove.